### PR TITLE
chore: better error handling

### PR DESCRIPTION
**Pull request by lucacasonato**

### CODE REVIEW

File: proxy/main.ts [hunk 1]
```diff
@@ -37,7 +46,16 @@ app.addEventListener("listen", (event) => {
 });
 
 app.addEventListener("error", (evt) => {
-  console.log("error", evt.error);
+  const ctx = evt.context;
+  if (ctx) {
+    const status = ctx.response.status;
+    console.log(
```
Comment (satyarohith) [inline proxy/main.ts:52]: nit: maybe use `console.error` for errors?

**Review by satyarohith — APPROVED** [review]
Looks good.

File: proxy/main.ts (hunk 1 above)
Comment (lucacasonato) [reply proxy/main.ts:52]: It doesn't render any differently - I personally think `console.error` is pretty useless :upside_down_face:

Comment (satyarohith) [reply proxy/main.ts:52]: I agree with the rendering part - I just thought it might be better to send errors to stderr for easy access.

Comment (lucacasonato) [reply proxy/main.ts:52]: Ah I see - didn't know we print errors to stderr :upside_down_face:.

**Review by ry — APPROVED** [review]

**Pull request merged**